app/views/pages/_schema_info.html.erb

```diff
@@ -1,205 +1,159 @@
-
-   NLM defines groups/arms this way:</p>
-   <ul class='regularDisplay'>
-     <li><i>Arm:</i> A pre-specified group or subgroup of participant(s) in a clinical trial assigned to receive specific intervention(s) (or no intervention) according to a protocol.</li>
-     <li><i>Group:</i> The predefined participant groups (cohorts) to be studied, corresponding to Number of Groups specified under Study Design (for single-group studies.</li>
-   </ul>
-
-   <p>In short, observational studies use the term ‘groups’; interventional studies use ‘arms’, though for the purpose of analysis, they both refer to the same thing. Because 'group' is more intuitive to the general public, AACT standardized on the term 'group(s)' and does not use the term 'arms'.</p>
-
-   <h4>Participant Groups: Registry vs Results</h4>
-   <p>When a study is registered in ClinicalTrials.gov, information is entered about how the study defines participant groups. In AACT, this information is stored in the <i>Design_Groups</i> table, while info about actual groups that is entered after the study has completed is stored in the <i>Result_Groups</i> table. (AACT has not attempted to link data between these 2 tables.)</p>
-
-   <p>Result information, for the most part, is organized in ClinicalTrials.gov by participant group. <i>Result_Contacts</i> & <i>Result_Agreements</i> are the only result tables not associated with groups. This section describes how AACT has structured group-related results data.</p>
-
-   <p>AACT provides four general categories of result information:</p>
-
-     <ul class='regularDisplay'>
-       <li>Participant Flow (Milestones & Drop/Withdrawals)</li>
-       <li>Baselines</li>
-       <li>Outcomes</li>
-       <li>Reported Events</li>
+  <section class="namingConventions" >
+    <h4 id='namingConventions'>Naming Conventions</h4>
+    <ul class='regularDisplay'>
+      <li>Table names are plural (e.g., <i>studies, facilities, interventions</i>).</li>
+      <li>Column names are singular (e.g., <i>description, phase, name</i>).</li>
+      <li>Multi-word names use underscores (e.g., <i>mesh_term, study_first_submitted_date</i>).</li>
+      <li>Case is ignored by PostgreSQL; <i>Studies</i>, <i>STUDIES</i>, and <i>studies</i> are equivalent.</li>
+      <li>Study design data from ClinicalTrials.gov use tables prefixed with <i>Design_</i>, distinguishing them from results tables (e.g., <i>Design_Groups</i> vs. <i>Result_Groups</i>).</li>
+      <li>Tables and columns use full names, avoiding abbreviations (e.g., <i>description</i> not <i>desc</i>).</li>
+      <li>Redundant wording is avoided (e.g., <i>Studies.source</i> instead of <i>Studies.study_source</i>).</li>
+      <li id='idInformation'>Columns ending with <i>_id</i> are foreign keys, linking the child table to the parent table's <i>id</i> column. For example:<br>
+        <i>Child_Table.parent_table_id = Parent_Table.id</i><br>
```

Comment:
We are not being consistent, we have a lot of documentation that uses all lower case naming but here we are using upper case. change it to `child_table.parent_table_id = parent_table.id`